### ISSUE TITLE
Error in reading data into a dataframe on encountering an empty cell

### ISSUE THREAD

**Issue by hepbc** [xlwings/xlwings#2307]
I have the following data in an xl sheet, cell A7 onwards

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Bhaskar/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Bhaskar/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

  | FY-5 |   | FY-4 | FY-3 | FY-2 | FY-1 | FY0
-- | -- | -- | -- | -- | -- | -- | --
INE002A01018 | 64852 | NULL | 85490 | 89193 | 80298 | 120991 | 142908
INE467B01029 | 32532 | NULL | 39512 | 42110 | 46547 | 53048 | 59259
INE982J01020 | NULL | NULL | -4343.8 | -2634.2 | -1763.4 | -2332.4 | -1631.5

</body>

</html>

I am reading this into a pandas dataframe using the following:
```
df = sheet.range('A7').options(pd.DataFrame, 
                                     header=1,
                                     index=False, 
                                     empty = "NA",
                                     expand='table').value
df

```
Output is:
<html>
<body>

NA | FY-5
-- | --
INE002A01018 | 64852.0
INE467B01029 | 32532.0
INE982J01020 | NULL

</body>
</html>

Basically, it stops reading the table on encountering an empty cell (C7). Setting the attribute "empty" to "NA" does not solve the problem. I cannot use the "used_range" attribute because this is just 1 table of many in the sheet.

Any ideas on what I can do to read the whole table correctly?

Thanks!
